cat/arm.cat

```diff
@@ -0,0 +1,67 @@
+ARM
+
+(* Model for ARM, ie with po-loc omitted from ppo *)
+include "cos.cat"
+
+(* Uniproc *)
+acyclic po-loc | rf | fr | co as uniproc
+
+(* Atomic *)
+empty rmw & (fre;coe) as atomic
+
+(* Utilities *)
+let dd = addr | data
+let rdw = po-loc & (fre;rfe)
+let detour = po-loc & (coe ; rfe)
+let addrpo = addr;po
+
+(*******)
+(* ppo *)
+(*******)
+
+(* Fences *)
+let dmb.st = po;[DMB.ST];po
+let dsb.st = po;[DSB.ST];po
+let dmb = po;[DMB];po
+let dmb.ish = po;[DMB.ISH];po
+let dsb = po;[DSB];po
+let isb = po;[ISB];po
+
+(* Initial value *)
+let ci0 = (ctrl; [ISB]) | detour
+let ii0 = dd | rfi | rdw
+let cc0 = dd | ctrl | addrpo (* po-loc deleted *)
```

Comment:
Why is `po-loc` removed?

Reply:
I copied the cat file [from herd7's repo](https://github.com/herd/herdtools7/blob/master/herd/libdir/arm.cat)

Reply:
I see.